Amplify/Categories/Auth/Request/AuthForgetDeviceRequest.swift

```diff
@@ -7,10 +7,13 @@
 
 import Foundation
 
+/// Request to forget a device of the user
 public struct AuthForgetDeviceRequest: AmplifyOperationRequest {
 
+    /// Device to forget
```

Comment:
It's optional--what's the behavior if not specified? (Presumably, forget current device?)